src/webapp/components/template-selector/TemplateSelector.tsx

```diff
@@ -286,14 +285,11 @@ export const TemplateSelector = ({
     const showPopulate = !(state.templateType === "custom" && !settings.showPopulateInCustomForms);
     const selected = state.id && state.templateId ? getOptionValue({ id: state.id, templateId: state.templateId }) : "";
 
-    const subSectionClasses = useMemo(() => ({ sectionHeader: classes.subSectionHeader, sectionPaper: classes.subSection }), [classes]);
+    const subSectionClasses = { sectionHeader: classes.subSectionHeader, sectionPaper: classes.subSection };
```

Comment:
[DONE] But here we do need the useMemo to keep it immutable, as the value is an object that's passed directly as prop.